src/Wizard.js

```diff
@@ -4,33 +4,39 @@ import React, { useState, useEffect } from 'react';
 import { findNextValid, findPreviousValid } from './utils';
 import { ControlsContext } from './Controls';
 import { StepContext } from './Step';
+import StateManager from './StateManager';
 
 export class ValidationError extends Error {}
 
 type Props = {
   onComplete: (currentStep: string) => void,
   children: React$Node,
-  debug?: Boolean,
+  debug?: boolean,
+  stateManager?: string,
 };
 
-const Wizard = ({ children, onComplete, debug }: Props) => {
+const Wizard = ({ children, onComplete, stateManager, debug }: Props) => {
   const [index, setIndex] = useState(0);
   const [steps, setSteps] = useState([]);
   const [isLoading, setLoadingState] = useState(false);
 
+  const href = typeof window === 'undefined' ? '' : window.location.href;
```

Comment:
Wizard shouldn't refer to window.location. I can't see any usages of `href` so perhaps you simply can remove this line.

Reply:
Unfortunately I need to make a reference to href in useEffect, or useEffect will not trigger.

Reply:
Adding it to the `useEffect` dependencies runs if `window.location.href` changes _between renders_.  However, a change to `window.location.href` _won't_ trigger a re-render, so a prop change must also happen for the `useEffect` to run. That can suddenly break if the prop change for some reason don't happen. I think it's safer to avoid having `href` as a dependency and solve the problem in a different way. Let's take a look at it together?

Reply:
Hrmm... Ok, I removed the reference to href again, and useEffect seems to run appropriately. ¯\_(ツ)_/¯